shotgun_api3/shotgun.py

```diff
@@ -1265,7 +1266,9 @@ def reset_user_agent(self):
         if self.client_caps.platform is not None:
             ua_platform = self.client_caps.platform.capitalize()
         self._user_agents = ["shotgun-json (%s)" % __version__,
-                             "Python %s (%s)" % (self.client_caps.py_version, ua_platform)]
+                             "Python %s (%s)" % (self.client_caps.py_version, ua_platform),
+                             "ssl-validation %s" % (int(self.config.no_ssl_validation is False))]
```

Comment:
I thought we talked about including the openssl version if available?

Would a string like "ssl 0.9.8a (validate)" make sense?  And if we're on an older Python that doesn't have access "ssl unknown (novalidate)" or something like that?